langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiStreamingLanguageModel.java

```diff
@@ -59,22 +59,36 @@ public class AzureOpenAiStreamingLanguageModel implements StreamingLanguageModel
     private OpenAIClient client;
     private final String deploymentName;
     private final Tokenizer tokenizer;
+    private final Integer maxTokens;
     private final Double temperature;
     private final Double topP;
-    private final Integer maxTokens;
+    private final Map<String, Integer> logitBias;
+    private final String user;
+    private final Integer n;
+    private final Integer logprobs;
+    private final Boolean echo;
+    private final List<String> stop;
     private final Double presencePenalty;
     private final Double frequencyPenalty;
+    private final Integer bestOf;
```

Comment:
According to [docu](https://learn.microsoft.com/en-us/azure/ai-services/openai/reference), seems that `bestOf` cannot be used with streaming

Reply:
Indeed! This is fixed